recipes/astartes-molecules/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: astartes-molecules
```

Comment:
Why is this needed? Making a whole package just for the license file seems odd.

Reply:
I had intended this as a metapackage (astartes has plugins that use aimsim), so the license file is there to indicate the metapackage is under the MIT license.

Reply:
Meta packages add an extra burden in our CIs and infrastructure and they can easily be just two lines on dependencies on users environments downstream. I would recommend to try at least a multiple-outputs package so this is not in separate feedstock. But even better would be to drop the metapackage.

Reply:
Got it, thanks for explaining! Will drop the metapackage